### DISCUSSION TITLE
Hybrid-car mode

### DISCUSSION THREAD

**Discussion by vicom147** [serkri/SmartEVSE-3#158] [Ideas]
Is it an idea to implement an Hybrid-car mode? 

So when you return after work, the car is charged with available solar power. 
But in the evening after sunset it charges in (what now is called) smart mode. 
So the battery is always fully charged at morning.

Or maybe easier to implement 2 timeframes during the day with different charging modes?

**Comment by dingo35** [comment]
I am leaning towards a "simple user mode", that has 3 charging deadline options, today, tomorrow, and later.

Here is someone who implemented it in HA:
https://gathering.tweakers.net/forum/list_message/75252394#75252394

**Reply by vicom147** [reply]
Will take a look at that, at the moment I don't have HA running.
